### PR TITLE
Improvements to the metrics service

### PR DESCRIPTION
**Pull request by sfc-gh-aivanou**
* Add `Makefile` with build targets
* Add SPCS metrics service spec
* Make prometheus work with mount storage
* Minor modifications to mdservice
* Add Readme

### CODE REVIEW

File: user-metrics/README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+The metrics stack consist of the following containers:
+* **Otel collector** [Otel Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib).
+* **Metadata Service**. Simple http metadata service that provides a list of compute pool endpoints. 
+If new compute pool is added, metadata service will discover all node that belong to it.
+* **Prometheus**. Prometheus framework that is mostly used to store metrics.
+* **Grafana**. Predefined dashboards that show compute pool, service and job metrics. 
+
+Metrics services consists of several containers defined above.
+Containers can be built manually from this repository, or one can use
+containers from the dockerhub: TODO: add link to all containers.
```
Comment (sfc-gh-tkommineni) [inline user-metrics/README.md:16]: https://hub.docker.com/u/snowflakedb

I will add mdservice to the same place

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 .idea
 spcs/examples/llama7b-chat-hf/snow/__pycache__
 spcs/examples/llama7b-chat-hf/spcs_spec.yaml
+.DS_Store
+user-metrics/prometheus/Dockerfile
```
Comment (sfc-gh-tkommineni) [inline .gitignore:5]: why are we adding prometheus dockerfile to gitignore?

Comment (sfc-gh-aivanou) [reply .gitignore:5]: yeah, it should be `user-metrics/metrics-service.yml`

**Review by sfc-gh-tkommineni — DISMISSED** [review]

File: user-metrics/README.md (hunk 1 above)
Comment (sfc-gh-tkommineni) [reply user-metrics/README.md:16]: I think these links can be added now

**Review by sfc-gh-tkommineni — APPROVED** [review]

**Pull request merged**